test/component/uk/gov/hmrc/organisationsdetailsapi/stubs/BaseSpec.scala

```diff
@@ -28,27 +33,62 @@ import play.mvc.Http.MimeTypes.JSON
 import scala.concurrent.duration.Duration
 
 trait BaseSpec
-  extends AnyFeatureSpec with BeforeAndAfterAll with BeforeAndAfterEach with GuiceOneServerPerSuite
+  extends AnyFeatureSpec with BeforeAndAfterAll with BeforeAndAfterEach with Matchers with GuiceOneServerPerSuite
     with GivenWhenThen {
 
   implicit override lazy val app: Application = GuiceApplicationBuilder()
     .configure(
+      "mongodb.uri"                             -> "mongodb://127.0.0.1:27017/organisations-details-api",
+      "microservice.services.integration-framework.host" -> "localhost",
```

Comment:
In order to comply with the latest auth upgrade please change the host to 127.0.0.1. This means that it routes through auth correctly as localhost will bypass.